### ISSUE TITLE
Can't append a Blob without a filename

### ISSUE THREAD

**Issue by jaydenseric** [octet-stream/form-data#13]
It should be possible to `.append()` a `Blob` instance to a `FormData` instance, without having to provide a third filename argument:

> The default filename for `Blob` objects is "blob".
> — https://developer.mozilla.org/en-US/docs/Web/API/FormData/append

This line assumes a filename exists:

https://github.com/octet-stream/form-data/blob/73d5f4dbffab6f6a27f26c6611e32662c98c2891/src/lib/FormData.mjs#L264

Which causes a Node.js `path.basename()` error:

```
The "path" argument must be of type string. Received undefined
```

**Comment by octet-stream** [comment]
Hi! Thank you for the feedback. I will take a look and fix it as soon as I can.

**Comment by octet-stream** [comment]
This should be fixed in v2.1.1
